### DISCUSSION TITLE
Replay crash on Loading Screen (CUSTOM/TOURNAMENT GAME)

### DISCUSSION THREAD

**Discussion by ViniPedrosoVP** [fraxiinus/ReplayBook#292] [Troubleshooting]
Hiya, ty for helping me!

I'm trying to open a replay file that a player from a tournament sent me so I can perform a live analysis.
This is the Replay File: https://drive.google.com/file/d/1GM0joRI1mb5PksbLg-kO2xKjobrq1cLC/view?usp=sharing

I downloaded the last patch (13.20) from the replay.xyz website, followed the installation procedures and indicated the path in the Replay Book. I tested it with my own replays of patch 13.20 and does it works! I tested another files from patch 13.20 and worked well too.

However, this replay in question (in addition to not showing the data correctly in the program - external bug already addressed) when opening on the loading screen, simply crashes and does not open. 

I'll leave the file in question here, and some screenshots.

I don't know if it's something related to this, but I noticed that the replay patch indicates 13.20.536.9576, but the client version available on the replay.xyz website is 13.20.535.2535 .  

Replay File: https://drive.google.com/file/d/1GM0joRI1mb5PksbLg-kO2xKjobrq1cLC/view?usp=sharing

![print](https://uploaddeimagens.com.br/images/004/649/059/original/print.png)
![print2](https://uploaddeimagens.com.br/images/004/649/061/full/screenshot2.png)
![print3](https://uploaddeimagens.com.br/images/004/649/062/full/screenshot2.png)

Edit: Even with language changed for EN_US (as i saw a tip from another user on discord). still have the same problems:
![print4](https://uploaddeimagens.com.br/images/004/649/066/full/screenshot3.png) 

-_

EDIT 2:
I Could open another files with no problem. My question now is:
There's any chance that CUSTOM /TOURNAMENT games are the problem?

**Comment by fraxiinus** [comment]
I wonder if it has something to do with the hotfix, I know 13.20 had a lot of issues on release. I unfortunately don't have a version of 13.20 that matches this replay so I can't check at all.
